tests/tracer/test_monkey.py

```diff
@@ -18,25 +17,25 @@ class TestPatching(SubprocessTestCase):
     @run_in_subprocess(env_overrides=dict())
     def test_patch_all_env_override_sqlite_none(self):
         # Make sure sqlite is enabled by default.
-        monkey.patch_all()
+        _monkey.patch_all()
```

Comment:
musing: I wonder if these tests should be testing `ddtrace.patch/patch_all()` instead of `_monkey.patch/patch_all()`.

This is not a blocking comment.